src/platforms/windows/daemon/windowstunnellogger.cpp

```diff
@@ -107,6 +108,10 @@ void WindowsTunnelLogger::process(int index) {
 
   QByteArray message((const char*)data + RINGLOG_TIMESTAMP_SIZE,
                      RINGLOG_MESSAGE_SIZE);
+  int nullIndex = message.indexOf(0x0);
+  if (nullIndex > 0) {
```

Comment:
`>=` ?

Reply:
No, Owen's suggestion is better.  The true root is trying to decode UTF16 as UTF8.  I'm just verifying that now.

Reply:
Conversion didn't work, so I made this change.